docs/setup_xcode.md

```diff
@@ -73,436 +44,351 @@ For reference, this is the version of Xcode we're using in this example.  Yours
 ```console
-$ make clean
-rm -rvf *.exe *~ *.out *.dSYM *.stackdump
 $ make main.exe
-g++ -Wall -Werror -pedantic -g --std=c++11 main.cpp stats.cpp p1_library.cpp -o main.exe
 $ ./main.exe
-enter a filename
-main_test_data.tsv
-enter a column name
-B
-...
+What's your name?
+Drew
+Hello Drew!
+```
+{: data-highlight="4" }
+
+Automate user input by putting it in a file.
+```
+Drew
 ```
+{: data-title="main_test.in" data-highlight="1" }
 
-If we put the user input in a file we can automate the user input.  We'll put it in a file called `main_test.in`.
+Redirect file `main_test.in` to stdin of `main.exe`.
 ```console
-$ cat main_test.in   # Peek at the contents of a file
-main_test_data.tsv
-B
-$ ./main.exe < main_test.in  # Redirect file content to main's stdin (cin)
-enter a filename
-enter a column name
-reading column B from main_test_data.tsv
-...
+$ ./main.exe < main_test.in
+What's your name?
+Hello Drew!
 ```
+{: data-highlight="1" }
 
 Without input redirection, here's how to type input in the Xcode command line.
 
 <img src="images/xcode255.png" width="768px" />
 
-Xcode no longer directly supports command-line style redirection.  One option is to simply type the input each time.  Another option is to use a work-around described in [https://gitlab.eecs.umich.edu/eecs281/wiki/wikis/xcode-file-redirection](https://gitlab.eecs.umich.edu/eecs281/wiki/wikis/xcode-file-redirection)
-
+Xcode does not support automated input redirection.  One option is to simply type the input each time.  Another option is to use a work-around described in [https://gitlab.eecs.umich.edu/eecs281/wiki/wikis/xcode-file-redirection](https://gitlab.eecs.umich.edu/eecs281/wiki/wikis/xcode-file-redirection)
 
-## Arguments and options
+### Arguments and options
 <div class="primer-spec-callout info" markdown="1">
-Skip this subsection for EECS 280 project 1.  You'll need it for project 2 and beyond.
+Skip this subsection for EECS 280 project 1.
 </div>
 
-*Arguments* and *options* are inputs to a program typed at the command line.  Arguments are often required.  Options (AKA *flags* or *switches*) start with a hyphen (`-`), and are typically optional.
-
-**Arguments example** from project 2:  `resize.exe` is the name of the program, and the arguments are `horses.ppm`,  `horses_400x250.ppm`, `400`, and `250`.
-```console
-$ ./resize.exe horses.ppm horses_400x250.ppm 400 250
-```
-{: data-variant="no-line-numbers" }
-
-**Options example** from project 5:  `main.exe` is the name of the program.  `train_small.csv` and  `test_small.csv` are arguments.  `--debug` is an option.
+Arguments and options are inputs to a program typed at the command line.  For example:
 ```console
 $ ./main.exe train_small.csv test_small.csv --debug
 ```
-{: data-variant="no-line-numbers" }
+{: data-variant="no-line-numbers" data-highlight="1" }
 
-To run a program with options or arguments in Xcode, select your scheme and then "Edit Scheme".
+- `main.exe` is the name of the program
```

Comment:
My guess is that these are files used in project 2? But it can be confusing to see these new filenames without context. Perhaps a bit of context stating "the files given in this example are from project X" would be helpful.

Reply:
c994e3d99802c88ee097db84ae37b60ba1b0a0c8